EC-84: Revise Configuration, Customers, eContent, Login and Logging (#5)
ezsystems/commerce-documentation: docs/guide/customers/customers_api/customer_profile_data_controller.md

# Customer profile data controller

`Silversolutions\Bundle\EshopBundle\Controller\CustomerProfileDataController`

## Actions

|Action|Route|Description|
|--- |--- |--- |
|`showDetailAction()`|`silversolutionsCustomerDetail`|Renders the profile detail page|

> |`addressBookAction()`|`silversolutions_address_book_list`|Renders the address book (a list with delivery addresses coming from ERP)|

Reviewer: ```suggestion
|`addressBookAction()`|`silversolutions_address_book_list`|Renders the address book (a list with delivery addresses that come from ERP)|
```

|`addressBookDeleteAction()`|`silversolutions_address_book_delete`|Removes the given delivery address from ERP and customer profile data|
|`logoutAction()`|`silversolutionsCustomerLogout`|Unsets all profile data within the session, logs out the user and redirects to the previous page|

## Profile detail page

![](../../img/customer_2.png)

> The user can manage their data profile data on this page.

Reviewer 2: ```suggestion
The user can manage their profile data on this page.
```

> In the address list the user can:

Reviewer: ```suggestion
The user can perform the following actions in the address list:
```

> - update existing delivery address

Reviewer: ```suggestion
- update an existing delivery address
```
